pkgs/games/anki/default.nix

```diff
@@ -31,7 +31,7 @@
 
 let
   # when updating, also update rev-manual to a recent version of
-  # https://github.com/dae/ankidocs
+  # https://github.com/ankitects/ankidocs
```

Comment:
dead link

Reply:
Oops, it's https://github.com/ankitects/anki-docs now, with a dash on the RHS.